editor/src/clj/editor/app_view.clj

```diff
@@ -2011,26 +2011,45 @@ If you do not specifically require different script states, consider changing th
       (ui/on-closed! stage (fn [_] (dispose-scene-views! app-view)))
       app-view)))
 
+(defn- make-info-box! []
+  (let [info-panel (HBox.)
+        left-label (Label. "This file is part of a library and cannot be saved.  ")
+        right-link (Hyperlink. "Read more…")
+        spacer (Region.)]
+    (HBox/setHgrow spacer Priority/ALWAYS)
+    (.setStyle info-panel "-fx-background-color: #444; -fx-padding: 4;")
+    (.setTextFill left-label Color/LIGHTGRAY)
+    (.setTextFill right-link Color/CORNFLOWERBLUE)
```

Comment:
We use CSS for styling, and prefer to use named overridable colors like `-df-defold-blue-light` for hyperlinks, `-df-text-dark` for hint text etc.

Reply:
Fixed!